apps/enc.c

```diff
@@ -418,15 +426,31 @@ int enc_main(int argc, char **argv)
                     BIO_printf(bio_err, "bad magic number\n");
                     goto end;
```

Comment:
It's not related to `PBKDF2` and do not impact arguments, but OK, I will prepare to enter a new discussion (not an easy task for me you know ! :-)